### ISSUE TITLE
mutmut command fails if no arguments are given

### ISSUE THREAD

**Issue by fmoor** [boxed/mutmut#8]
running the mutmut cli without arguments prints a trace back. I realize that this is not how it is supposed to be used but it is not very friendly. Perhaps the help text could be printed.

Windows 10 Enterprise
Python 3.6.2 (v3.6.2:5fd33b5, Jul 8 2017, 04:14:34) [MSC v.1900 32 bit (Intel)] on win32
mutmut (0.0.3)

**Comment by boxed** [comment]
It is how it should be used actually. Can you supply the stack trace so I can have a look?

**Comment by fmoor** [comment]
```
Traceback (most recent call last):
  File "C:\Users\fmoor\.venv\barometer\Scripts\mutmut-cmd.py", line 185, in <module>
    main()
  File "c:\users\fmoor\.venv\barometer\lib\site-packages\click\core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\fmoor\.venv\barometer\lib\site-packages\click\core.py", line 697, in main
    rv = self.invoke(ctx)
  File "c:\users\fmoor\.venv\barometer\lib\site-packages\click\core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\fmoor\.venv\barometer\lib\site-packages\click\core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "c:\users\fmoor\.venv\barometer\lib\site-packages\mutmut\__init__.py", line 35, in wrapper
    f(*args, **kwargs)
  File "C:\Users\fmoor\.venv\barometer\Scripts\mutmut-cmd.py", line 56, in main
    paths_to_mutate = [x.strip() for x in paths_to_mutate.split(',')]
AttributeError: 'NoneType' object has no attribute 'split'
```

**Comment by boxed** [comment]
Ah, I realized I only thought out how this should work and never actually coded it up :(

When I have you on the subject: where is the code located that you want to mutate?

**Comment by fmoor** [comment]
The command that I ran to get the stack trace is below. This is a Django project so I have test directories in the app directories i.e. `barometer\app\tests\`

Note that I had to change `C:\Users\fmoor\.venv\barometer\Scripts\mutmut.py` to `C:\Users\fmoor\.venv\barometer\Scripts\mutmut-cmd.py` to get the script to execute.

```
C:\Users\fmoor\projects\barometer>mutmut-cmd
```

**Comment by boxed** [comment]
And the app code is all under C:\Users\fmoor\barometer? I can't make sense of your directory structure based on what I have so far :P

**Comment by fmoor** [comment]
The relevant parts look like this:
```
barometer\
  <app name>\
    tests\
    ...
```

**Comment by boxed** [comment]
That doesn't make sense given the stack traces. Do you have the project inside the virtualenv directory?

**Comment by fmoor** [comment]
No. The project is not in the virtualenv directory.On Sep 1, 2017 9:50 PM, Anders Hovmöller <notifications@github.com> wrote:That doesn't make sense given the stack traces. Do you have the project inside the virtualenv directory?

**Comment by boxed** [comment]
In any case, I've pushed a fix for this and some new code that tries to intelligently guess where your code to mutate is. 

I've also released 0.0.14 with fixes for all your issues (hopefully). Hope it works for you now!
